docs/source/reference/artifacts.rst

```diff
@@ -4,7 +4,13 @@ optuna.artifacts
 ================
 
 The :mod:`~optuna.artifacts` module provides the way to manage artifacts (output files) in Optuna.
-Please note that methods defined in each ArtifactStore are not intended to be directly accessed by library users.
+Please check :ref:`artifact_tutorial` for the instruction.
```

Comment:
I do not know why this line does not appear in the doc published by CI:
- https://optuna--5501.org.readthedocs.build/en/5501/reference/artifacts.html

![image](https://github.com/optuna/optuna/assets/47781922/b165411f-dd18-4205-aaf5-883604fca722)

Plus, the link for `remove_for_artifact_store` was not updated as well.
It appears as `FAQ` in the doc.

Reply:
Was a cache issue
It was resolved once I forced to reload it!